docs/pages/accordion.md

```diff
@@ -165,6 +198,6 @@ For example, <a target="_blank" href="#deeplink3">http://example.com/#deeplink3<
 
 When linking directly to a accordion panel, it might not be obvious that the content appears within a accordion panel. An additional attribute `data-deep-link-smudge` rolls the page up slightly after deep linking (to a horizontal accordion) so that the accordion is at the top of the viewport.
 
-```html_example
+```html
```

Comment:
So style sherpa is authoritative for Foundation documentation?  Asking for clarification, as I can alter my own submission to reflect it if that's the case.